### ISSUE TITLE
Fails to load in 0.20

### ISSUE THREAD

**Issue by clintongormley** [jprante/elasticsearch-skywalker#3]
Hiya Jorg

I get this error when using 1.0.1 with v 0.20 of elasticsearch:

```
[09:55:08,323][WARN ][plugins                  ] [Mantis] plugin skywalker, failed to invoke custom onModule method
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:601)
        at org.elasticsearch.plugins.PluginsService.processModule(PluginsService.java:152)
        at org.elasticsearch.plugins.PluginsModule.processModule(PluginsModule.java:61)
        at org.elasticsearch.common.inject.Modules.processModules(Modules.java:64)
        at org.elasticsearch.common.inject.ModulesBuilder.createInjector(ModulesBuilder.java:58)
        at org.elasticsearch.node.internal.InternalNode.<init>(InternalNode.java:157)
        at org.elasticsearch.node.NodeBuilder.build(NodeBuilder.java:159)
        at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:67)
        at org.elasticsearch.bootstrap.Bootstrap.main(Bootstrap.java:200)
        at org.elasticsearch.bootstrap.ElasticSearchF.main(ElasticSearchF.java:15)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:601)
        at org.tanukisoftware.wrapper.WrapperSimpleApp.run(WrapperSimpleApp.java:290)
        at java.lang.Thread.run(Thread.java:722)
Caused by: java.lang.NoClassDefFoundError: org/elasticsearch/action/support/BaseRequestBuilder
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:791)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:423)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:356)
        at org.elasticsearch.plugin.skywalker.SkywalkerPlugin.onModule(SkywalkerPlugin.java:45)
        ... 19 more
Caused by: java.lang.ClassNotFoundException: org.elasticsearch.action.support.BaseRequestBuilder
        at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:423)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:356)
        ... 32 more
```

ta

clint

**Comment by jprante** [comment]
Yes, I need to rebuild against 0.20. Currently only 0.19 is available...

**Comment by jgagnon1** [comment]
+1

**Comment by jprante** [comment]
Just released 1.1.0, built against the 0.20 branch of Elasticsearch.
